lib/bigtable/cache/redisClient.js

```diff
@@ -64,14 +68,33 @@ class RedisClient {
 
     this.redis.on("end", () => debug("Redis connection has ended."));
   }
+
+  async setLastAccess(keyOrKeys) {
+
+    if (this.errorEconnrefused) {
```

Comment:
errorEconnrefused is that camelcase? :P errorEconnRefused or errorECONRefused

Reply:
should be ECONNREFUSED, thought I just made all of them lowered cased and camel case with error just in case we need to catch another error